feature-group-definitions/webtransport.yml

```diff
@@ -1,3 +1,43 @@
 spec: https://w3c.github.io/webtransport/
 caniuse: webtransport
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/3472
+compat_features:
+  - api.WebTransport
+  - api.WebTransport.WebTransport
+  - api.WebTransport.WebTransport.options_allowPooling_parameter
+  # As of 2023-08-10, the spec says "feature at risk"
+  # - api.WebTransport.WebTransport.options_congestionControl_parameter
+  - api.WebTransport.WebTransport.options_requireUnreliable_parameter
+  - api.WebTransport.WebTransport.options_serverCertificateHashes_parameter
+  - api.WebTransport.byob_readers
```

Comment:
BYOB reader support came in Chrome 109, but I think it's reasonable to say WebTransport was supported since Chrome 97. I'd suggest commenting these out.